pyscriptjs/src/runtime.ts

```diff
@@ -17,17 +17,25 @@ const logger = getLogger('pyscript/runtime');
 
 export type RuntimeInterpreter = PyodideInterface | null;
 
-export type RuntimeConfig = {
-    src: string;
-    name?: string;
-    lang?: string;
-};
-
 export type AppConfig = {
-    autoclose_loader: boolean;
     name?: string;
+    description?: string;
     version?: string;
+    type?: string;
```

Comment:
Not sure we need the type on the app config here tbh. An App is an app and the other type of project we thought (plugin) is not going to have a config, is it?

Reply:
With 5 second of more thinking... to avoid complications, let's keep it for now. Too many variables.

Reply:
type could be plugin/extension too..

Reply:
keeping it and resolving